companies/octo-telematics.json

```diff
@@ -0,0 +1,28 @@
+{
+  "slug": "octo-telematics",
+  "relevant-countries": [
+      "it",
+      "es",
+      "de",
+      "gb",
+      "pl",
+      "us",
+      "br"
+  ],
+  "categories": [
+      "telecommunication",
+      "commerce"
+  ],
+  "name": "Octo Group s.p.a.",
+  "address": "Via Vincenzo Lamaro 51\n00173 Roma\nItalia",
```

Comment:
If the company is only relevant to the country it is based in, we write the country in that country's language. Otherwise (like in this case, where it is relevant to multiple countries), we always fallback to English.